WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/details/ThreatDetailsViewModelTest.kt

```diff
@@ -0,0 +1,61 @@
+package org.wordpress.android.ui.jetpack.scan.details
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import kotlinx.coroutines.InternalCoroutinesApi
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.mockito.ArgumentMatchers.anyLong
+import org.mockito.Mock
+import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.test
+import org.wordpress.android.ui.jetpack.scan.details.ThreatDetailsViewModel.UiState
+import org.wordpress.android.ui.jetpack.scan.details.ThreatDetailsViewModel.UiState.Content
+import org.wordpress.android.ui.jetpack.usecases.GetThreatModelUseCase
+
+@InternalCoroutinesApi
+class ThreatDetailsViewModelTest : BaseUnitTest() {
+    @Mock private lateinit var getThreatModelUseCase: GetThreatModelUseCase
+    @Mock private lateinit var builder: ThreatDetailsListItemsBuilder
+    private lateinit var viewModel: ThreatDetailsViewModel
+    private val threatId = 1L
+
+    @Before
+    fun setUp() {
+        viewModel = ThreatDetailsViewModel(getThreatModelUseCase, builder)
+    }
+
+    @Test
+    fun `given threat id, then on start, threat details are retrieved`() = test {
```

Comment:
Minor (🔍): The action of the test sentence probably need to be replaced with `when` (instead of `then`). In the end you might want something like the: `given threat id, when on start is triggered, then threat details are retrieved`

Let me know your thoughts. 🙏  

PS: The same apply for the test below: `given threat id, when on start is triggered, then ui is updated with content`

Reply:
Oh yes, thanks for noticing 😅. Rephrased in 2811298.